### PR TITLE
Use EventType and EventSource from Core

### PR DESCRIPTION
**Pull request by kevinlind**
## Description

Set Mobile Core dependency to `2.2.0-SNAPSHOT`.
Update local uses of `com.adobe.eventType.edgeMedia`, `com.adobe.eventSource.createTracker`, and `com.adobe.eventSource.trackMedia` to those in Core EventType and EventSource. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: code/edgemedia/build.gradle [hunk 1]
```diff
@@ -67,14 +67,11 @@ dependencies {
     testImplementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.1'
     //noinspection GradleDependency
 
-    testImplementation 'org.json:json:20180813'
```
Comment (kevinlind) [inline code/edgemedia/build.gradle:70]: This dependency didn't appear to be used so I removed it.

File: code/edgemedia/build.gradle [hunk 2]
```diff
@@ -67,14 +67,11 @@ dependencies {
...
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
     androidTestImplementation "com.adobe.marketing.mobile:edge:$mavenEdgeVersion"
     androidTestImplementation "com.adobe.marketing.mobile:edgeidentity:2.+"
-    androidTestImplementation "com.fasterxml.jackson.module:jackson-module-kotlin:2.12.7"
```
Comment (kevinlind) [inline code/edgemedia/build.gradle:75]: Was getting error while running Android tests "Cannot fit requested classes in a single dex file (# methods: 70537 > 65536)", so I removed this dependency after creating a workaround.

File: code/edgemedia/src/androidTest/java/com/adobe/marketing/mobile/util/JsonTestUtils.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+package com.adobe.marketing.mobile.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.Map;
+
+public class JsonTestUtils {
+
+    @SuppressWarnings("unchecked")
+    public static Map<String, Object> parseNetworkResponseAsMap(String body) {
```
Comment (kevinlind) [inline code/edgemedia/src/androidTest/java/com/adobe/marketing/mobile/util/JsonTestUtils.java:21]: Created this Java class as a workaround to the build error "Cannot fit requested classes in a single dex file (# methods: 70537 > 65536)".  By using the Jackson Databind ObjectMapper in Java, I could remove the dependency on `jackson-module-kotlin` and therefore reduce the number of methods.

**Review by addb — APPROVED** [review]
Looks great! 💯

**Pull request merged**